openapi.yaml

```diff
@@ -16,175 +16,511 @@ host: api.oregonstate.edu
+          description: Successfully update a Game
+          schema:
+            $ref: '#/definitions/GameResult'
+          headers:
+            Location:
+              type: string
+              format: url
+              description: Location of the updated game
         '400':
-          description: Bad request
+          description: Invalid game object
           schema:
             $ref: '#/definitions/Errors'
         '500':
           description: Internal server error
           schema:
             $ref: '#/definitions/Errors'
-  /pets/{id}:
-    get:
-      summary: Info for a specific pet
+    delete:
       tags:
-        - pets
-      description: Get pet by unique ID
-      operationId: getPetById
+      - games
+      summary: "Remove a game specified by its id"
+      operationId: removeGameByID
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
       parameters:
+        - name: gameID
+          in: "path"
+          description: "Id of game to be removed"
+          required: true
+          type: "integer"
+      responses:
+        204:
+          description: "game successfully removed"
+        404:
+          description: "No game with this id"
+          schema:
+            $ref: "#/definitions/Error"
+        500:
+          description: "Internal server error"
+          schema:
+            $ref: "#/definitions/Error"
+  /games/{gameID}/players:
+    get:
+      summary: Get a list of players in the game
+      tags:
+        - players
+      description: get players by game id
+      operationId: getPlayersByGameID
+      parameters: 
         - $ref: '#/parameters/authorization'
-        - name: id
+        - name: gameID
           in: path
-          description: Unique Pet ID
+          description: unique integer
+          required: true
+          type: integer
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/PlayerResults'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+  /players/{playerID}:
+    get:
+      summary: Info for a specific player in a game
+      tags:
+        - players
+      description: get player by player id
+      operationId: getPlayerByID
+      parameters: 
+        - $ref: '#/parameters/authorization'
+        - name: playerID
+          in: path
+          description: g[int]p[int] where the first int is the ID of the game and the second int is the ID of the player in that game.
           required: true
           type: string
+          pattern: '^g[1-9]\d*p[1-9]\d*$'
       responses:
         '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PetResult'
+            $ref: '#/definitions/PlayerResult'
```

Comment:
Based on your definition, here is an example of the returned PlayerResult:

```json
{
  "links": {
    "self": "string"
  },
  "data": {
    "id": "g1p1",
    "type": "player",
    "attributes": {
      "players": [
        {
          "cards": [
            {
              "attributes": {
                "cardNumber": "A",
                "cardSuit": "Diamond"
              }
            }
          ],
          "bet": 0,
          "status": "called"
        }
      ]
    }
  }
}
```

Should the `players` array be included in a single player object?